app/controllers/api/v1/courses_controller.rb

```diff
@@ -5,6 +5,11 @@ def index
         @courses = Course.all.includes(:sites, :provider, :site_statuses)
         paginate json: @courses
       end
+
+      def show_by_recruitment_cycle
```

Comment:
`show` should be used when retrieving a single `Course`. Also, I don't think we need a separate method for this, we should be able to simply point this endpoint to `index`, and once we actually have more than one recruitment year's worth of data, we'll sort out the filtering then.

Reply:
`show_by_recruitment_cycle` that can be removed